Scripts/MonoBehaviourService.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace DUCK.Utils
+{
+	/// <summary>
+	/// Allows access to MonoBehaviour like events to non MonoBehaviours
+	/// </summary>
+	public sealed class MonoBehaviourService
+	{
+		private static MonoBehaviourService instance;
+		public static MonoBehaviourService Instance { get { return instance ?? (instance = new MonoBehaviourService()); }}
+
+		/// <summary>
+		/// OnUpdate is called every frame.
+		/// </summary>
+		public event Action OnUpdate;
+
+		/// <summary>
+		/// OnLateUpdate is called every frame at the end of the
+		/// <a href="https://docs.unity3d.com/Manual/ExecutionOrder.html">Game Logic Update</a>
```

Comment:
Why is this the only one with a link?

Reply:
I dunno, I copied it from the original. Didn't you make this ?

Reply:
It's minor anyway. Wanted it noted for the record and yes I made the the original.